### ISSUE TITLE
increase max Items per page to 24

### ISSUE THREAD

**Issue by minusf** [allinurl/goaccess#2100]
Would it be possible to increase the maximum `Items per page` under `Panels` in the html report to `24` (currently it's `20`)?  That way it would be possible to see the `Time distribution` without paging.  Thank you.

**Comment by allinurl** [comment]
I agree, would you be able to submit a PR with this change? The file that needs to change is [**this**](https://github.com/allinurl/goaccess/blob/master/resources/tpls.html#L255). Thanks
